product_docs/docs/pgd/5.9/upgrades/upgrade_paths.mdx

```diff
@@ -11,22 +11,26 @@ You can upgrade from any version 5.x release to a later 5.x release.
 
 ## Upgrading from version 4 to version 5
 
-Upgrades from PGD 4 to PGD 5 are supported from version 4.3.0. For earlier
-versions, upgrade to 4.3.0 before upgrading to 5. See [Upgrading within
-4](/pgd/4/upgrades/upgrade_paths/#upgrading-within-version-4) for more
-information. 
-
-Generally, we recommend that you upgrade to the latest version 4
-release before upgrading to the latest version 5 release. After upgrading to
-4.3.0 or later, the following upgrade paths are possible.
-
-| From version | To version |
-| ---- | -- |
-| 4.3.0 | 5.0.0 or later |
-| 4.3.1 | 5.1.0 or later |
-| 4.3.2 | 5.1.0 or later |
-| 4.3.3 | 5.1.0 or later |
-| 4.3.7 | 5.7.0 or later |
+Generally, we recommend that you upgrade to the latest version 4 release
+before upgrading to the latest version 5 release.
```

Comment:
I'd state "before upgrading to version 5 release". Otherwise, one could read this as "not necessary for older v5 releases than the latest". Nit-picking, but let's try to get this as clear as possible.